client/src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,24 @@
+
+.Navbar {
```

Comment:
Should this navbar stuff be in a different branch?

Reply:
Hrm. It should be. I might have used git add -A by accident.